lib/api/consumers/api-consumer.js

```diff
@@ -29,16 +29,28 @@ export default async function apiConsumers({data: {dataType, jobType, data, stat
   done()
 }
 
-const addressConsumer = async (jobType, addresses, statusID) => {
-  const addressesCount = addresses.length
-  const addressesValidation = await checkAddresses(addresses, jobType)
-  if (addressesValidation.isValid) {
+const addressConsumer = async (jobType, payload, statusID) => {
+  const checkRequestData = async (payload, jobType) => {
```

Comment:
Est-ce qu'il ne faudrait pas sortir cette fonction en dehors de addressConsumer ?

Reply:
Why not. A voir lors de la refacto. 🤷‍♂️